assets/docs/neso613/ASR_TFLite/scripts/1.md

```diff
@@ -0,0 +1,10 @@
+# Lite neso613/ASR_TFLite/scripts/1
+
+TF Lite quantized version of the Conformer to generate text from audio.
+
+<!-- parent-model: neso613/ASR_TFLite/1 -->
+<!-- asset-path: legacy -->
+
+[![Open Colab notebook]](https://colab.research.google.com/drive/1RN91rBO5UkOlKk_pc9kVivtjgP76aeox?usp=sharing)
```

Comment:
Is this page actually a separate model? Or is it just a colab link? If the colab demonstrates using the ASR_TFLite/pre_trained_models/English model, then you can put this link directly on that markdown page.

Reply:
This colab demonstrates the end to end inference of TFLite model. So, should I put this on same ASR_TFLite/pre_trained_models/English/1.md?